Makefile

```diff
@@ -7,6 +7,9 @@ ARTIFACT_DIR := $(if $(ARTIFACT_DIR),$(ARTIFACT_DIR),tests/test_results)
 TEST_TAGS := $(if $(TEST_TAGS),$(TEST_TAGS),"")
 SUITE_ID := $(if $(SUITE_ID),$(SUITE_ID),"nosuite")
 
+EVAL_MODEL := $(if $(EVAL_MODEL),$(EVAL_MODEL),"gpt")
+EVAL_THRESHOLD := $(if $(EVAL_THRESHOLD),$(EVAL_THRESHOLD),"0.5")
```

Comment:
default this in the test code and be done with it, i don't think we need a way to control it from outside the test code.